### ISSUE TITLE
Loss graph for training

### ISSUE THREAD

**Issue by abeyang00** [AlexeyAB/darknet#546]
Is there a way to look at loss graph during training so i can stop the training myself instead of waiting til the end iteration?

**Comment by fjutsisejc** [comment]
You can train in windows OS. There is a loss graph.

**Comment by AlexeyAB** [comment]
There is Loss graph for training by default if you use Windows or Linux (with GUI desktop: GNOME, KDE, Xfce)

**Comment by abeyang00** [comment]
does it by default show on terminal during training? or is there a special command that needs to be put in for before executing ./darknet detector train ?

**Comment by fjutsisejc** [comment]
AlexeyAB‘s darknet is default .

**Comment by Caroline1994** [comment]
@AlexeyAB Will the loss graph saved automatically after the training?I was use your repo

**Comment by AlexeyAB** [comment]
@Caroline1994 You should click on Window with loss-chart and press `s` button.

I just added fix to store loss-chart at the end of training when `(current_batch== max_batches)` where `max_batches=` is param in your cfg-file: https://github.com/AlexeyAB/darknet/blob/c52fa4742866125e35e88d0653dac99473e5fffb/cfg/yolov3.cfg#L20

**Comment by dfsaw** [comment]
In which location will the chart be saved? Also can we plot chart for Validation set?
